src/tactic/equiv_rw.lean

```diff
@@ -91,6 +95,7 @@ meta def equiv_congr_lemmas : tactic (list expr) :=
 do exprs ←
   [
   `equiv.of_iff,
+  `category_theory.iso.to_equiv,
```

Comment:
I don't have a good feeling how many functors are `functorial`.  Is `functorial list` an instance?

If lots of things are `functorial`, then I think putting this lemma first will cause quite a bit of a slowdown because every subterm will be visited with both `≃` and many times with `≅`.  That is, the number of steps would be quadratic instead of linear if you do not pass an isomorphism to `equiv_rw`.

Reply:
I'm pretty sure this is going to be fine, but I'm going to come back to this after `transport`, and I have some better examples of usage of this change.

I've marked this PR as WIP for now.